common/src/main/java/com/wynntils/screens/characterselector/CharacterSelectorScreen.java

```diff
@@ -112,6 +114,24 @@ protected void doInit() {
                 (int) addButtonHeight,
                 this));
 
+        float disconnectButtonWidth = Texture.DISCONNECT_BUTTON.width() * currentTextureScale;
+        float disconnectButtonHeight = Texture.DISCONNECT_BUTTON.height() / 2f * currentTextureScale;
+        this.addRenderableWidget(new DisconnectButton(
+                (int) (this.width - disconnectButtonWidth - 10f),
+                (int) (10f),
```

Comment:
Notice how every other Y coordinate is based on height while this is hard coded, meaning it won't work on all resolutions. Please fix this

Reply:
Ok, i will, but isn't the PlayButton using basically the same thing?

Reply:
Also isn't the X also affected by it?

Reply:
Fixed with the commit [781dde9](https://github.com/Wynntils/Artemis/pull/1650/commits/781dde999d7fd791393e8b0548cab6e3bf7990c7)